docs/API/V1/list.md

```diff
@@ -0,0 +1,118 @@
+# `GET /api/v1/datasets/list`
+
+This API returns an `application/json` document describing the set of datasets
+accessible to the client. (An unauthenticated client can only access "public"
+datasets.)
+
+The list of datasets may be further filtered by owner, access, name substring,
+or by creation date range using the query parameters.
+
+## Query parameters
+
+`access`    string \
+Select whether only `private` or only `public` access datasets will be included
+in the list. By default, all datasets readable by the authenticated user are
+included. For example, without constraints `/datasets/list` for an authenticated
+user will include all `public` datasets plus all datasets owned by the
+authenticated user; specifying `private` will show only the authenticated user's
+private datasets, while specifying `public` will show only `public` datasets
+(regardless of ownership).
+
+`end` date/time \
+Select only datasets created on or before the specified time. Time should be
+specified in ISO standard format, as `YYYY-MM-DDThh:mm:ss.ffffff[+|-]HH:MM`.
+If the timezone offset is omitted it will be assumed to be UTC (`+00:00`); if
+the time is omitted it will be assumed as midnight (`00:00:00`) on the
+specified date.
+
+`limit` integer \
+"Paginate" the selected datasets by returning at most `limit` datasets. This
+can be used in conjunction with `offset` to progress through the full list in
+smaller chunks either to assist with a paginated display or to limit data
+transfer requirements.
+
+`metadata` list \
+Request the named [metadata](../metadata.md) to be returned along with the
+resource name and ID. This can be a single metadata key path or comma-separated
+list of such strings. The `metadata` query parameter can be repeated. For
+example, the following are all equivalent:
+* `?metadata=dataset.created,server.deletion,user`
+* `?metadata=dataset.created&metadata=dataset.deletion,user`
+* `?metadata=dataset.created&metadata=dataset.deletion&metadata=user`
+
+`offset` integer \
+"Paginate" the selected datasets by skipping the first `offset` datasets that
+would have been selected by the other query terms. This can be used with
+`limit` to progress through the full list in smaller chunks either to assist
+with a paginated display or to limit data transfer requirements.
+
+`owner` string \
+The `username` property of a registered user. Only datasets owned by that user
+will be included in the list.
```

Comment:
The descriptions of the other parameters begin with a verb. Perhaps something like

> Select only datasets owned by the specified username.